prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -271,7 +271,7 @@ void rxr_pkt_handle_readrsp_sent(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_en
 	assert(tx_entry->window >= 0);
 	if (tx_entry->bytes_sent < tx_entry->total_len) {
 		if (efa_mr_cache_enable && rxr_ep_mr_local(ep))
```

Comment:
Doesn't this also need a `rxr_ep_is_cuda_mr` check?

Reply:
This is the emulated Read path, when GPU Buffer is used, we really should not have go there. I think I will add an assertion here about the mr cannot be a cuda_mr.

Reply:
Got it, thanks.